packages/eslint-plugin-va/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "eslint-plugin-va",
```

Comment:
@timwright12 I noticed you used the `vagov` convention in the names of our other npm packages. I kept the `va` convention here so that we don't have to rename the disabled rules in a bunch of files in `vets-website`. Is this fine, or should we change this?

Reply:
that's fine